source/extensions/common/wasm/wasm.cc

```diff
@@ -504,6 +504,22 @@ void replaceHeader(Http::HeaderMap* map, absl::string_view key, absl::string_vie
     map->addCopy(lower_key, std::string(value));
 }
 
+const uint8_t* decodeVarint(const uint8_t* start, const uint8_t* end, uint32_t* out) {
```

Comment:
Nit: since we're moving the pointer, `pos` might be slightly better name than `start`. Also, should this be `const`?

Reply:
I'll change start->pos.  What should be const?  This is a free function and all the arguments are const except the out variable.

Reply:
I was asking if `start` should be `const` (like in current version), since we're modifying it.

Reply:
So, since it is, we are good AFAICT.